internal/types/path.go

```diff
@@ -17,17 +17,87 @@ package types
 import (
 	"fmt"
 	"strconv"
+	"strings"
 )
 
+// Path represents field path type. It should be used wherever we work with paths or dot notation.
```

Comment:
```suggestion
// The path represents the field path type. It should be used wherever we work with paths or dot notation.
```

Reply:
No, doc comment must start from the exact word used as type/var/function etc. So, in this particular case it's still just `Path` :)

Reply:
> No, doc comment must start from the exact word used as type/var/function etc. So, in this particular case it's still just `Path` :)

Changed.